### DISCUSSION TITLE
Allow adding images to notes from gallery

### DISCUSSION THREAD

**Discussion by jgscherber** [streetcomplete/StreetComplete#4650] [Feedback]
When creating notes, adding an image is only allowed from the camera. This doesn't allow taking a picture and then adding it to a note after. 

The note dialog should allow adding notes from the gallery also

**Comment by matkoniecz** [comment]
See https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ#Can_I_choose_photos_from_the_gallery_for_uploading_to_notes.2C_instead_of_using_camera.3F
